user/supertux2/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "supertux2"
+pkgver = "0.6.3"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCMAKE_BUILD_TYPE=RELEASE",
```

Comment:
the canonical name is Release, it may not matter but might as well write it properly